documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -34,7 +34,14 @@ Consider a shorter interval for fast changing clusters, although this increases
 [discrete]
 == Contents of optimization proposals
 
-The following table describes the properties contained in an optimization proposal:
+An optimization proposal consists of two main sections: the _summary_ and the _broker load_.
```

Comment:
This still looks like the old status only PR, it should read something like:

"An optimization proposal consists of two main sections: the _summary_ which is stored in the status of the KafkaRebalance custom resource and the _broker load_ which is stored in a ConfigMap."